schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    'id' int,
+    'name' varchar,
+    'date_of_birth' date,
+    'escape_attempts' int,
+    'neutered' boolean,
+    'weight_kg' decimal
+);
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
- [OPTIONAL] I'll suggest you make the `id` column, a primary key of the table, not NULL and auto-increment, this will make sure each row has a unique ID and you don't have to input it yourself. :+1: You can achieve this in two ways. :+1: :smile: 
- - Adding these two extra lines
```sh
id int NOT NULL IDENTITY(1,1)
PRIMARY KEY (id)
```
- - Or changing the ID line into this
```sh
id SERIAL PRIMARY KEY,
```